lib/MusicBrainz/Server/Form/Utils.pm

```diff
@@ -207,7 +207,10 @@ sub validate_username {
     if (defined $username) {
         unless (defined $previous_username && $editor_model->are_names_equivalent($previous_username, $username)) {
             my $sanitized_name = sanitize($username);
-            if ($username ne $sanitized_name) {
+            if (
+                $username ne $sanitized_name ||
+                $sanitized_name =~ qr{://}
```

Comment:
"Check for consecutive spaces" doesn't make sense for this error, does it :)

Reply:
It does for genuine mistyping. All occurrences of `://` are ill-intentioned; See the IRC backlogs.

Reply:
I know, but if some artist or whatnot tries to make their homepage their username, they'll be very confused :)